README.md

```diff
@@ -66,14 +66,29 @@ Screen.poly(20, 50, 20, 100)
 Screen.flush()
 ```
 
-Built-in properties to access the environment, like time or links:
+Built-in properties to access the environment, like counter or time:
 
 ```python
+counter = Env.counter
 current_time = Env.time
-link_count = Env.link_count
```

Comment:
I believe it can create confusion from people that are coming from mlog. I certainly saw that, thought about how it is `@links` in mindustry and my brain automaticaly striked out possibility to write anything that starts with `@` using `Env`. It took me to dive into source to figure out that compiler would happily write anything after `Env.` as variable that starts with `@` instad of just what is defined stuff in pyndustri.pyi and enforce it.